numpyro/infer/initialization.py

```diff
@@ -30,7 +30,7 @@ def init_to_median(site=None, num_samples=15):
             return init_to_uniform(site)
 
 
-def init_to_prior(site=None):
+def init_to_sample(site=None):
```

Comment:
init_to_prior -> init_to_sample

Reply:
This seems reasonable since `init_to_median` is an alternative way to use the prior. I guess we could be even more verbose with `init_to_prior_sample`, `init_to_prior_median`, but I'm not sure other options. Maybe `init_to_generated`?